### DISCUSSION TITLE
Pretrained Model Release Roadmap

### DISCUSSION THREAD

**Discussion by lucasjinreal** [jinfagang/yolov7#4] [Announcements]
I planned to release pertained models in the next few month. It's not easy to train all supported models. I still encourage community can help train **any version model you interested in** and PR to main repo.

- [ ] DETR coco;
- [ ] AnchorDETR coco;
- [ ] YOLOX-s;
- [ ] DETR ONNX;
- [ ] DETR tensorrt;
- [ ] YOLOv7-Mask;
- [ ] more

**Comment by amirhp110** [comment]
Would you please some sample code on how to train pretrained model. The command option --opts MODEL.WEIGHTS [pre/trained-model-name.pth does not work for me.

Another question:
Training on my laptop with RTX 3080 is taking 3 days, expected, but training the same on machine with 8 x A100 40GB, is taking 1.8 days!
Do I need to change any parameters to take advantage of whole power? Only 5 GB of each 8 GPU is being utilized!

I am following Aarohi's post to train custom dataset, so let me know if any yaml file needs changes for better performance. I did increase the batches and didn't help.

**Comment by lucasjinreal** [comment]
@amirhp110  Just 

```
python train_det.py --config-file configs/coco/exp/yolo2go_mobileone_exp1.yaml --num-gpus 2 --resume
```

**Reply by amirhp110** [reply]
Thanks for quick response 
Don't we need to somehow provide the pre-trained model name?
Like in yolov7 (the other gir repo) the yolov7.pt is provided in the command or some yaml config.
